components/Footer/Footer.js

```diff
@@ -0,0 +1,78 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function Footer() {
+  return (
+    <>
+      <StyledFooterList>
+        <StyledIconContainer>
+          <StyledButton>
+            <Image
+              width={40}
+              height={40}
+              src={"/assets/icons8-calendar-24.png"}
+            />
+          </StyledButton>
+          <p>Schedule</p>
+        </StyledIconContainer>
+        <StyledIconContainer>
+          <StyledButton>
+            <Image
+              width={40}
+              height={40}
+              alt="bookmark"
+              src={"/assets/icons8-bookmark-outline.svg"}
+            />
+          </StyledButton>
+          <p>bookmark</p>
+        </StyledIconContainer>
+      </StyledFooterList>
+    </>
+  );
+}
+
+const StyledFooterList = styled.ul`
+  list-style-type: none;
+  width: 100vw;
+  display: flex;
+  margin: 0;
```

Comment:
It might make sense to set the margin and the padding in the GlobalStyles to 0 for certain elements.